ssh/proxy.go

```diff
@@ -105,10 +105,9 @@ func (p *ProxyConn) handleAuthMsg(msg *userAuthRequestMsg, proxyConf *ProxyConfi
 
 		for _, signer := range signers {
 			msg, err = p.signAgain(username, msg, signer)
-			if err != nil {
-				break
+			if err == nil {
+				return msg, nil
 			}
-			return msg, nil
```

Comment:
I fix this logic. Because the previous logic break the loop when even one index has failed.
Now, loop is not stop until signAgain return err not nil.
If all signers all failed, get out the loop and break case automaticly.